modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -0,0 +1,204 @@
+// TELCODOCS-213 4.8 Using Assisted Installer to Install & Manage ocp Clusters on Bare Metal
+//
+//
+//
+
+[id=ztp-ai-install-ocp-clusters-on-bare-metal_{context}"]
+
+.Enabling Assisted Installer Service on Bare Metal
+
+The Assisted Installer Service (AIS) deploys {product-title} clusters. Red Hat Advanced Cluster Management (RHACM) 2.3+ ships with AIS, which is deployed when the MultiClusterHub Operator is enabled on the RHACM hub cluster.
+
+RHACM 2.3+ only supports *Single Node OpenShift (SNO)* cluster deployments. A SNO cluster runs OpenShift Container Platform (ocp) on top of a single BMO node. When the host is provisoned, the single node cluster acts as both supervisor and worker node.
+
+.Enable assisted installer service
+
+.Prerequisites
+
+. OpenShift Container Platform 4.8+\{product-version}
```

Comment:
. Install {product-title} {product-verstion} on a hub cluster.
. Install {rh-acm} and create the `MultiClusterHub` resource.